Makefile

```diff
@@ -99,27 +98,16 @@ all ::
 	@echo "PDF built: `ls -lAFgh $(BUILD_DIR)/manual/build/latex/nexus-fairmat.pdf`"
 
 $(BASE_CLASS_DIR)/%.nxdl.xml : $(BASE_CLASS_DIR)/$(NYAML_SUBDIR)/%.yaml
-	nyaml2nxdl $<
-	mv $(BASE_CLASS_DIR)/$(NYAML_SUBDIR)/$*.nxdl.xml $@
+	nyaml2nxdl $< --output-file $@
 
 $(CONTRIB_DIR)/%.nxdl.xml : $(CONTRIB_DIR)/$(NYAML_SUBDIR)/%.yaml
-	nyaml2nxdl $<
-	mv $(CONTRIB_DIR)/$(NYAML_SUBDIR)/$*.nxdl.xml $@
+	nyaml2nxdl $< --output-file $@
 
 $(APPDEF_DIR)/%.nxdl.xml : $(APPDEF_DIR)/$(NYAML_SUBDIR)/%.yaml
-	nyaml2nxdl $<
-	mv $(APPDEF_DIR)/$(NYAML_SUBDIR)/$*.nxdl.xml $@
-
-NXDLS := $(NXDL_APPDEF) $(NXDL_CONTRIB) $(NXDL_BC)
-nyaml :
```

Comment:
don't we need this make command to generate the yaml files

Reply:
Yes, we do but I moved it to the second file.